profiles/source_profile.go

```diff
@@ -429,12 +431,80 @@ func NewSourceProfileConnection(source string, params map[string]string) (Source
 	return conn, nil
 }
 
+type DirectConnectionConfig struct {
+	Host     string `json:"host"`
+	User     string `json:"user"`
+	Password string `json:"password"`
+	Port     string `json:"port"`
+	DbName   string `json:"dbName"`
+}
+
+type DatastreamConnProfile struct {
+	Name     string `json:"name"`
+	Location string `json:"location"`
+}
+
+type DataflowConfig struct {
+	Location      string `json:"location"`
+	Network       string `json:"network"`
+	Subnetwork    string `json:"subnetwork"`
+	HostProjectId string `json:"hostProjectId"`
+}
+
+type DataShard struct {
+	DataShardId           string                `json:"dataShardId"`
+	SrcConnectionProfile  DatastreamConnProfile `json:"srcConnectionProfile"`
+	DestConnectionProfile DatastreamConnProfile `json:"destConnectionProfile"`
+	DataflowConfig        DataflowConfig        `json:"dataflowConfig"`
+	TmpDir                string                `json:"tmpDir"`
+	StreamLocation        string                `json:"streamLocation"`
+	LogicalShards         []LogicalShard        `json:"databases"`
+}
+
+type LogicalShard struct {
+	DbName         string   `json:"dbName"`
+	TableInclude   []string `json:"tableInclude"`
```

Comment:
If we're excluding tables, we would probably not create schema for them as well. In this case, this belongs in the session file.

Reply:
Yes, I had initially thought of including this here as a stop gap solution for selecting tables to migrate until individual table migrations came, I have removed this now.
Will update this logic with information derived from the session file with the individual table migrations feature.